From: SeongJae Park <sj@kernel.org>
Date: Wed, 29 Jan 2025 10:21:11 -0800
Subject: [PATCH] mm/damon/core: fix set but not used 'nr_regions' in
 damon_get_intervals_adaptation_bp()

Refer to Closes: link for detail.

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202501300041.usRaByZc-lkp@intel.com/
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/core.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/mm/damon/core.c b/mm/damon/core.c
index eecf7c2281b9..52184f53aefe 100644
--- a/mm/damon/core.c
+++ b/mm/damon/core.c
@@ -1338,11 +1338,10 @@ static unsigned long damon_get_intervals_adaptation_bp(struct damon_ctx *c)
 {
 	struct damon_target *t;
 	struct damon_region *r;
-	unsigned long nr_regions = 0, access_samples = 0, score_bp;
+	unsigned long access_samples = 0, score_bp;
 	unsigned long adaptation_bp;
 
 	damon_for_each_target(t, c) {
-		nr_regions += damon_nr_regions(t);
 		damon_for_each_region(r, t)
 			access_samples += r->nr_accesses;
 	}
-- 
2.39.5

